corehq/apps/userreports/reports/util.py

```diff
@@ -88,7 +106,9 @@ def data_rows(self):
             if column.visible:
                 column_ids.extend(column_id_to_expanded_column_ids.get(column.column_id, [column.column_id]))
 
-        return [[raw_row[column_id] for column_id in column_ids] for raw_row in self.get_data()]
+        data = self.get_data()
+
+        return [[raw_row[column_id] for column_id in column_ids] for raw_row in data]
```

Comment:
why was this change needed?
I guess it was for debugging purposes.

Reply:
https://github.com/dimagi/commcare-hq/pull/32964/commits/1d6f6b5d1098d16c6685c2cfafa5c13b1bc18bcd